Source/SwiftLintFramework/Rules/Style/FileTypesOrderRule.swift

```diff
@@ -112,10 +125,14 @@ public struct FileTypesOrderRule: ConfigurationProviderRule, OptInRule {
         let dict = file.structureDictionary
         return dict.substructure.filter { substructure in
             guard let declarationKind = substructure.declarationKind else { return false }
-            guard !substructure.inheritedTypes.contains("PreviewProvider") else { return false }
+
+            let hasExcludedInheritedType = substructure.inheritedTypes.contains { inheritedType in
```

Comment:
I would prefer a method for this check that can also be used in line 174.

Reply:
moved to a private extension